src/init.gradle

```diff
@@ -39,6 +39,15 @@ rootProject {
         excludedModuleIds.set(excludedModuleIdsProperty)
     }
 
+    def dependencyTree = tasks.register("dependencyTree", DependencyTree) {
+        mustRunAfter(aggregatedText)
+        Configuration config = project.getConfigurations().findByName("allCodeCoverageReportClassDirectories")
```

Comment:
Right, but it doesn't really make sense. It's normal that most (if not all) projects don't have a `runtimeClasspath` for the root project since it's not a library but the parent project. It doesn't even make sense to include in the report.